src/main/routing/router_queue_codel.c

```diff
@@ -197,12 +197,10 @@ static Packet* _routerqueuecodel_dequeueHelper(QueueManagerCoDel* queueManager,
 }
 
 static CSimulationTime _routerqueuecodel_controlLaw(guint count, CSimulationTime ts) {
-    CSimulationTime newTS = ts + CODEL_PARAM_INTERVAL_SIMTIME;
-
-    double result = ((double)newTS) / sqrt((double)count);
+    // control_law: t + (INTERVAL / sqrt(count));
+    double result = ((double)CODEL_PARAM_INTERVAL_SIMTIME) / sqrt((double)count);
     double rounded = round(result);
-
-    return (CSimulationTime)rounded;
+    return ts + ((CSimulationTime)rounded);
```

Comment:
Is the goal to be exactly the same as the reference? If so, should it just cast the float to an integer instead of rounding?